README.md

```diff
@@ -37,6 +37,10 @@ Default is `Unknown`.
 
 * `DOCKER_PASSWORD` - Docker registry account password
 
+* `KLAR_INSECURE` - Allow Klar to access registries with bad SSL certificates. Default is `false`.
```

Comment:
It is better to remove the code about insecure registry, and this PR should only focus on json output.